tutorials/send-emails-tem-serverless-framework/index.mdx

```diff
@@ -0,0 +1,105 @@
+---
+meta:
+  title: Sending emails with Transactional Email and Serverless using the Serverless Framework
+  description: Discover how to send emails with Transactional Email and Serverless using the Serverless Framework
+  h1: Sending emails with Transactional Email and Serverless using the Serverless Framework
+  paragraph: Discover how to send emails with Transactional Email and Serverless using the Serverless Framework
+tags: transactional-email serverless emails functions serverless-framework
+categories:
+  - transactional-email
+  - functions
+dates:
+  validation: 2024-01-15
+  posted: 2024-01-15
+---
+
+In this tutorial, you will learn how to send emails with [Transactional Email](/managed-services/transactional-email/quickstart/) and [Serverless Functions](/serverless/functions/quickstart/) using the [Serverless Framework tool](/serverless/functions/reference-content/deploy-function/#serverless-framework).
+
+<Macro id="iam-requirements" />
+ 
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [configured your domain with Transactional Email and added SPF, DKIM and MX records](/managed-services/transactional-email/quickstart/)
+  - You have [installed the Scaleway CLI](https://github.com/scaleway/scaleway-cli) on your machine
+</Message>
+
+## Install and clone the necessary tools
+
+1. Open a terminal and paste the following command to install the Serverless Framework CLI on your local machine:
+
+        ```
+        npm install serverless -g
+        ```
+2. Clone the GitHub repository containing the configuration files needed to deploy your function:
+        ```
+        git clone !!!ADD REPO HERE!!
+        ```
+
+3. Access the repository and paste the following command to install the [Serverless plugin]():
+        ```
+        cd tem && serverless plugin install -n serverless-scaleway-functions
+        ```
+4. Copy and paste the following commands to export your environment variables. Make sure you add your **own variables**.
+        ```
+        export SCW_ACCESS_KEY=<Scaleway access key>
+        export SCW_SECRET_KEY=<Scaleway secret key>
+        export SCW_DEFAULT_ORGANIZATION_ID=<Scaleway default Organization>
+        export SCW_DEFAULT_PROJECT_ID=<Scaleway default Project>
+        export SCW_DEFAULT_REGION="fr-par"
+        export SCW_API_URL="https://api.scaleway.com"
+        ```
+
+## Add your variables
+
+1. In the `tem` repository, open the `serverless.yml` file and replace `$SCW_SECRET_KEY`, `$SCW_PROJECT_ID`, `$SCW_REGION` in the `provider` section with your own variables.
```

Comment:
For secret management this is not the recommanded way to do it, we encourage to use env var here: https://github.com/scaleway/serverless-scaleway-functions/blob/master/docs/secrets.md  and make the same change in the file under this comment 😇